ijPlugin/src/main/resources/org/jetbrains/research/tasktracker/ui/main/panel/template/agreement.html

```diff
@@ -0,0 +1,38 @@
+<div class="container">
```

Comment:
<img width="496" alt="Screenshot 2023-10-30 at 09 15 29" src="https://github.com/JetBrains-Research/tasktracker-3/assets/8993915/dfcadf06-bdd4-4b32-b3ba-594d669490a3">
Could you please make it prettier?
1) add a margin between the form description and itself (1)
2) Add a header before a list of agreements, like "I agree to the terms and conditions:"
3) put "Optional" or "Required" right after checkbox to indicate which checkboxes are required and optional
4) make the margin between agreements smaller (2) 
5) put the agreements to the left part of the screen, not in the center
6) put a mock link in at least one of the examples

Reply:
Also, add an error popup to inform the user that the user has to agree to all required terms and conditions + personal info to start (in this state)
<img width="480" alt="Screenshot 2023-10-30 at 09 21 26" src="https://github.com/JetBrains-Research/tasktracker-3/assets/8993915/7ff9297b-e523-4d03-9a70-81ca94c589fa">